gooddata-sdk/gooddata_sdk/catalog/workspace/service.py

```diff
@@ -78,6 +86,49 @@ def delete_workspace(self, workspace_id: str) -> None:
             )
         self._entities_api.delete_entity_workspaces(workspace_id)
 
+    def get_declarative_workspace(self, workspace_id: str) -> CatalogDeclarativeWorkspaceModel:
+        return CatalogDeclarativeWorkspaceModel.from_api(self._layout_api.get_workspace_layout(workspace_id))
+
+    def put_declarative_workspace(self, workspace_id: str, workspace: CatalogDeclarativeWorkspaceModel) -> None:
+        self._layout_api.put_workspace_layout(workspace_id, workspace.to_api())
+
+    def get_declarative_workspaces(self) -> CatalogDeclarativeWorkspaces:
+        return CatalogDeclarativeWorkspaces.from_api(self._layout_api.get_workspaces_layout())
+
+    def put_declarative_workspaces(self, workspace: CatalogDeclarativeWorkspaces) -> None:
+        self._layout_api.set_workspaces_layout(workspace.to_api())
+
+    def store_declarative_workspaces(self, path: Path) -> None:
+        workspaces = self._layout_api.get_workspaces_layout()
+        workspace_data_filters = workspaces.workspace_data_filters
+        with open(path / "workspace_data_filters.yaml", "w+") as fp:
+            yaml.safe_dump([w.to_dict() for w in workspace_data_filters], fp, indent=2, sort_keys=True)
+        for workspace in workspaces.workspaces:
+            workspace_path = path / f"{workspace.id}.yaml"
+            with open(workspace_path, "w+") as fp:
+                yaml.safe_dump(workspace.to_dict(), fp, indent=2, sort_keys=True)
+
+    def load_and_put_declarative_workspaces(self, path: Path) -> None:
+        workspace_data_filters_path = path / "workspace_data_filters.yaml"
+        if not workspace_data_filters_path.is_file():
+            raise ValueError(f"Path {workspace_data_filters_path} is not valid.")
+        workspace_files_path = path.glob("*.yaml")
+        if not workspace_files_path:
+            raise ValueError(f"There are no .yaml files in {workspace_files_path}.")
+        with open(workspace_data_filters_path, "r") as f:
+            workspace_data_filters = yaml.safe_load(f)
+        workspaces = []
+        for workspace in workspace_files_path:
+            if workspace.stem != "workspace_data_filters":
+                with open(workspace, "r") as f:
+                    workspaces.append(yaml.safe_load(f))
+
+        self.put_declarative_workspaces(
+            CatalogDeclarativeWorkspaces.from_api(
+                {"workspaces": workspaces, "workspace_data_filters": workspace_data_filters}
+            )
+        )
+
```

Comment:
May I kindly ask you for adding methods for managing ldm and analytics models?
There is really strong use case behind.